nix/sources.json

```diff
@@ -1,14 +1,14 @@
 {
     "nixpkgs": {
-        "branch": "release-24.11",
+        "branch": "master",
         "description": "Nix Packages collection & NixOS",
         "homepage": "https://github.com/NixOS/nixpkgs",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "31571599597cf5bd035928576b6abe922451ce7d",
-        "sha256": "1m4b3bgbch8isxq01h6ghz61jrfwf7cvp13srxgsin7jxm6hrxiw",
+        "rev": "ca4a1e055eaf8051cfc5b299f3a01ec16f2fde58",
+        "sha256": "0pp6bhvbbh5mz7fhg7ajnin4ak6411ry78mrnmhykgyp9nkdhj0c",
         "type": "tarball",
-        "url": "https://github.com/NixOS/nixpkgs/archive/31571599597cf5bd035928576b6abe922451ce7d.tar.gz",
+        "url": "https://github.com/NixOS/nixpkgs/archive/ca4a1e055eaf8051cfc5b299f3a01ec16f2fde58.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     }
 }
```

Comment:
this was the first commit/branch i could find that supported a kind recent enough (0.27.0) to include kubernetes versions that have the volume expansion test fixed.  I understand if this is too "bleeding edge" to pull in.  But if the tests pass -- at least the commit and sha256 are correct

Reply:
@addyess See #82 for that We will take care of that there